components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -3333,6 +3333,17 @@ func TestCreateRule(t *testing.T) {
 			_, err := storage.NewRule("new-id-1", projID, "name", storage.Node, []storage.Condition{})
 			assert.Error(t, err)
 		},
+		"creating a condition with zero entries returns an error": func(t *testing.T) {
```

Comment:
Suggest explicitly say "for 'equals' operator" in this test,
and add another test for the member-of operator.